taskfiles/lint.yaml

```
@@ -826,25 +829,31 @@ tasks:
         {{.ROOT_DIR}}/tools/yscope-dev-utils/exports/lint-configs/.rustfmt.toml \
         {{.ROOT_DIR}}/.rustfmt.toml
 
-  rust-workspace-fmt:
+  # Runs `cargo +nightly fmt` in the root Rust workspace directory with the specified flags.
+  #
+  # @param {string} CARGO_FMT_FLAGS The flags to pass to the `cargo +nightly fmt` command.
+  cargo-workspace-fmt:
     internal: true
-    deps: ["rust-lint-configs", ":toolchains:rust"]
-    dir: "{{.ROOT_DIR}}"
     requires:
-      vars: ["RUSTFMT_FLAGS"]
+      vars: ["CARGO_FMT_FLAGS"]
+    dir: "{{.ROOT_DIR}}"
+    deps: ["rust-lint-configs", "toolchains:rust"]
     cmd: |-
       . "$HOME/.cargo/env"
-      cargo +nightly fmt --all -- {{.RUSTFMT_FLAGS}}
+      cargo +nightly fmt --all -- {{.CARGO_FMT_FLAGS}}
```

```suggestion
      "$HOME/.cargo/bin/cargo" +nightly fmt --all -- {{.CARGO_FMT_FLAGS}}
```
Not really important but does this work? If so and you'd like to update, there are plenty other similar cases.